Tests/ClrAssembly/ClrAssembly.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>net40;net45;netcoreapp2.0</TargetFrameworks>
-    <PlatformTarget Condition="'$(TargetFramework)' == 'netcoreapp2.0'">x64</PlatformTarget>
+    <PlatformTarget Condition="'$(TargetFramework)' == 'netcoreapp2.0' AND '$(OS)' != 'Windows_NT'">x64</PlatformTarget>
```

Comment:
Not fully sure about this change, but I think that x86 Windows is supported by .NET Core.

Reply:
The problem is specifying Any CPU, that doesn't work.

Reply:
I can't understand why it doesn't work. It doesn't work on Mono? AnyCPU is the default in the dotnet sdk and it usually works.

Reply:
Do we even need `PlatformTarget` in `ClrAssembly`? Doesn't it only affect the dummy assembly which we don't use?